config/routes.rb

```diff
@@ -3,7 +3,7 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
   resources :systems, :auth, :translations, :drafts, :resources, :custom_pages, :languages, :attributes,
-            :translated_attributes, :pages, :views, :attachments, :translated_pages, :follow_ups
+            :translated_attributes, :pages, :views, :attachments, :translated_pages, :follow_ups, :resource_types
```

Comment:
I think it's OK like that.  404 is the response if an unimplemented method is called